src/ClampHelper.sol

```diff
@@ -12,19 +12,22 @@ abstract contract ClampHelper is AssertHelper {
     function clampBetween(
         uint256 value,
         uint256 low,
-        uint256 high
+        uint256 high,
+        bool enableLogs
```

Comment:
this should be a global setting. we don't want to change the API from the one in `crytic/properties`

Reply:
Fixed by adding ClampWrapper